src/style.css

```diff
@@ -1 +1,9 @@
 /* styles go here */
+.form_field {
+  margin-bottom: 10px;
```

Comment:
Most possibly the reason why your form doesn't pass the tests in our pipeline is bc you add margin bottom also to last elements in each fieldset. Try to use some css selectors to exclude last child from each fieldset from this css rule.